js/scripts.js

```diff
@@ -15,7 +15,9 @@ jQuery(document).ready(function ($) {
         dataslide = $(this).attr('data-slide');
 
         if (direction === 'down') {
-            $('.navigation li[data-slide="' + dataslide + '"]').addClass('active').prev().removeClass('active');
+            //$('.navigation li[data-slide="' + dataslide + '"]').addClass('active').prev().removeClass('active');
+            $('.navigation li[data-slide="' + dataslide + '"]').removeClass('active');
+			$('.navigation li[data-slide="' + dataslide + '"]').next().addClass('active');

```

Comment:
and `$('.navigation li[data-slide="' + dataslide + '"]')` could be cached instead of query-selecting it each line